src/sidebar/store/modules/selection.js

```diff
@@ -163,9 +163,24 @@ const setTab = (newTab, oldTab) => {
   };
 };
 
+/**
+ * Return state object representing a reset selection. This
+ * resets user-set filters (but leaves focus mode intact)
+ *
+ * @return {Object}
+ */
+const resetSelection = () => {
+  return {
+    filterQuery: null,
```

Comment:
Since there are no params, `resetSelection` could just be a value (`emptySelection` ?) rather than a function.

Reply:
I need to be sure I'm getting a new object reference every time. Not sure I grok...?